src/modules/packages/main.py

```diff
@@ -146,42 +162,48 @@ def run_operations(pkgman, entry):
             else:
                 pkgman.install(entry[key])
         elif key == "try_install":
-            # we make a separate package manager call for each package so a single
-            # failing package won't stop all of them
+            # we make a separate package manager call for each package so a
+            # single failing package won't stop all of them
             for package in entry[key]:
                 if isinstance(package, str):
                     try:
                         pkgman.install([package])
                     except subprocess.CalledProcessError:
-                        libcalamares.utils.debug("WARNING: could not install package {}".format(package))
+                        warn_text = "WARNING: could not install package "
+                        libcalamares.utils.debug(warn_text + package)
```

Comment:
Perhaps better to do :

warn_text = "foo"
warn_text += package
debug(warn_text)